python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -3,11 +3,13 @@
 ## @file jiminy_py/log.py
 
 import numpy as np
+from pandas import read_csv
```

Comment:
I don't like the idea to add pandas as a dependency for only this feature, which is able on the standard library `csv`.